build.gradle.kts

```diff
@@ -34,6 +34,9 @@ dependencies {
     implementation("org.apache.commons:commons-compress:1.21") {
       because("previous transitive version 1.20 pulled from Apache POI 5.0.0 has CVE")
     }
+    implementation("org.apache.santuario:xmlsec:2.2.3") {
```

Comment:
This is a transitive dependency via Apace POI (again).  There is no later version of POI.